c_api/impl/AuxIndexStructures_c.h

```diff
@@ -126,8 +126,24 @@ int faiss_RangeSearchPartialResult_set_lims(
 int faiss_RangeSearchPartialResult_new_result(
     FaissRangeSearchPartialResult* res, idx_t qno, FaissRangeQueryResult** qr);
 
+
+FAISS_DECLARE_CLASS(DistanceComputer)
+int faiss_DistanceComputer_new(FaissDistanceComputer** p_dc);
+int faiss_DistanceComputer_set_query(FaissDistanceComputer *dc, const float *x);
+int faiss_DistanceComputer_vector_to_query_dis(FaissDistanceComputer *dc, idx_t i, float *qd);
+/** 
+ * Compute distance between two stored vectors
+ *
+ * @param dc          opaque pointer to distance computer object
+ * @param key         output distance
+ **/
+int faiss_DistanceComputer_symmetrik_dis(FaissDistanceComputer *dc, idx_t i, idx_t j, float *vd);
```

Comment:
It's `symmetric_dis` in the main API, it might be better to stick to the same name.

```suggestion
int faiss_DistanceComputer_symmetric_dis(FaissDistanceComputer *dc, idx_t i, idx_t j, float *vd);
```

Reply:
My bad... a typo